app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -54,35 +54,38 @@ def current_journey_routing_name
   end
 
   def process_one_login_authentication_callback(auth)
-    onelogin_user_info_attributes = auth.info.to_h.slice(
-      *SignInForm::OneloginUserInfoForm::ONELOGIN_USER_INFO_ATTRIBUTES.map(&:to_s)
-    )
+    onelogin_user_info = auth.info.to_h.slice("email", "phone")
+    onelogin_credentials = auth.credentials
 
-    journey_session.answers.assign_attributes(onelogin_user_info: onelogin_user_info_attributes)
+    journey_session.answers.assign_attributes(
```

Comment:
We really need to implement a `journey_session.answers.update!` method that does this two step process for us 😂